### PR TITLE
Bug/1435-fix-overwrite-is-being-evaluated-after-heavy-calculations-are-executed

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#1435
OP#1436

### CODE REVIEW

**Review by randy-seng — COMMENTED** [review]
Looks good to me! Just a few comments that I have~

File: OTVision/transform/transform.py [hunk 1]
```diff
@@ -126,40 +135,41 @@ def main(
...
+        tracks_utm_df = transform(
+            tracks_px=tracks_px_df,
+            homography=homography,
+            refpts_utm_upshifted_predecimal_pt1_1row=refpts_utm_upshift_predecimal,
+            upshift_utm=upshift_utm,
+        )
+        log.info("Tracks transformed")
+
+        # Add crs information tp metadata dict
+        metadata_dict["trk"]["utm"] = True
```
[the remark this answers is not in the document]
Comment (martinbaerwolff) [reply OTVision/transform/transform.py:153]: Not done for now, as it is not MVP. I marked this as a #TODO in the code.

File: OTVision/convert/convert.py [hunk 2]
```diff
@@ -206,7 +218,7 @@ def convert(
 
     elif input_filetype != ".h264":
         raise TypeError("Input video filetype has to ne .h264")
```
Comment (randy-seng) [inline OTVision/convert/convert.py:220]: Typo
```suggestion
        raise TypeError("Input video filetype has to be .h264")
```

**Review by randy-seng — DISMISSED** [review]
Looks good to me! Just found a typo along the way~

**Review by randy-seng — APPROVED** [review]

**Pull request merged**